### ISSUE TITLE
About the number of the raw_data frame

### ISSUE THREAD

**Issue by JCZ404** [yanglei18/Mix-Teaching#2]
Hi, Thanks for your excellent work and code release. I followed the construction to download the raw_data of the KITTI dataset and extracted the image frames via the provided script `convert_det_format.py`, but finally, I only got 46612 image frames, which is not the unlabeled image number 48K mentioned in your paper. Is that correct? or I missed some details?

**Comment by Senwang98** [comment]
@Zhangjiacheng144 
Hello, how do you download raw kitti data?
It seems that there is no way to use `wget` to download data in China now.

**Comment by yanglei18** [comment]
@Senwang98 You need to configure VPN environment.

**Comment by yanglei18** [comment]
@Zhangjiacheng144 The 48K mentioned in paper is not exact. We collect the KITTI raw scenes consisting of 156 sequences. Then, the samples that appear in KITTI 3D validation and test set in continuous sequences are excluded to avoid data leakage. The remaining **35K** samples are used as unlabeled data for semi-supervised training.

**Comment by JCZ404** [comment]
Thanks for your kind reply, where is the part about conducting the data filtering in your code? There is only the code to download the KITTI raw data, could you please provide the complete data preparation code(mainly for the filtering of the overlapped data with validation and test dataset) to reproduce the performance? Thanks!

**Comment by destinyls** [comment]
@Zhangjiacheng144 Thanks for your attention, we have uploaded the code to help filter images overlapped with validation and test set.

**Comment by JCZ404** [comment]
Got it, thanks!

**Comment by JCZ404** [comment]
@yanglei18 Hi, sorry bother you again because I can't find the scripts in your code to filter images overlapped with validation and test set, could you tell me where is these files? BTW, I'm confused whether we need to filter the overlapped images with test set, since as the author of KITTI dataset claimed in their paper, they already remove the video sequence of test set from the raw data. Thanks ahead!  
![image](https://user-images.githubusercontent.com/51013927/232955629-1821deb4-3b29-4ddd-9ecf-973f5fd1d6c9.png)

**Comment by yanglei18** [comment]
@JCZ404 You can  refer to lines 112~115 in "raw_data_tools/convert_det_format.py" to help filter the overlapped images with the training set.

**Comment by JCZ404** [comment]
Ok, thanks!
